cafeteria/src/main/java/org/itt/utility/CafeteriaApp.java

```diff
@@ -0,0 +1,680 @@
+package org.itt.utility;
+
+import org.itt.controller.*;
+import org.itt.model.*;
+
+import java.math.BigDecimal;
+import java.sql.SQLException;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class CafeteriaApp {
+    private final AuthController authController;
+    private final MenuController menuController;
+    private final FeedbackController feedbackController;
+    private final NotificationController notificationController;
+    private final RecommendationController recommendationController;
+    private final RecommendationEngine recommendationEngine;
+    private final OrderController orderController;
+    private final ProfileController profileController;
+    private final Scanner scanner;
+
+    public CafeteriaApp() {
+        this.authController = new AuthController();
+        this.menuController = new MenuController();
+        this.feedbackController = new FeedbackController();
+        this.notificationController = new NotificationController();
+        this.recommendationController = new RecommendationController();
+        this.recommendationEngine = new RecommendationEngine();
+        this.orderController = new OrderController();
+        this.profileController = new ProfileController();
+        this.scanner = new Scanner(System.in);
+    }
+
+    public void run() {
+        try {
+            System.out.print("Enter User ID: ");
```

Comment:
Let's look at making functions smaller while also making sure SRP is being followed.